charts/etcd/templates/etcd-statefulset.yaml

```diff
@@ -182,6 +182,12 @@ spec:
         {{- if .Values.sharedConfig.autoCompactionRetention }}
         - --auto-compaction-retention={{ .Values.sharedConfig.autoCompactionRetention }}
         {{- end }}
+{{- end }}
+{{- if .Values.backup.etcdSnapshotTimeout }}
+        - --etcd-snapshot-timeout={{ .Values.backup.etcdSnapshotTimeout }}
+{{- end }}
+{{- if .Values.backup.etcdDefragTimeout }}
+        - --etcd-defrag-timeout={{ .Values.backup.etcdDefragTimeout }}
```

Comment:
`values.yaml` would contain the default values of these flags (i.e `8m`)?

Reply:
yes

Reply:
Maybe I'm looking at it the wrong way, but do we need to have those values in `values.yaml`? We can depend on `etcdbr` to provide default values to these flags, so adding here too might be redundant. In that case, these values need not be required to be in `values.yaml`.

Although idk, could be added as a safety check if you think it might be necessary. WDYT?

Reply:
yes, default values can be from `etcdbr`. We are settings default values in `values.yaml` it means that we can also change them other than default values. 
I will prefer to have the 2nd line of defence in `values.yaml` rather than depending on values coming from `etcdbr`.
And it’s better to keep `values.yaml` UpToDate on adding the additional flags otherwise values.yaml keep lagging behind .

Reply:
Thanks for explaining :)
Updated to include values in `values.yaml`. Now when values are not provided in the etcd crd, values from `values.yaml` are passes